gensim/models/tfidfmodel.py

```diff
@@ -39,24 +41,28 @@ def resolve_weights(smartirs):
 
     Returns
     -------
-    3-tuple (local_letter, global_letter, normalization_letter)
+    str of (local_letter, global_letter, normalization_letter) or None
```

Comment:
What does `None` result mean?

Reply:
Confuse the reader. I removed the polymorphism in 35f0f9db.